### PR TITLE
Create responsive

### PR DESCRIPTION
**Pull request by Hana-latif98**

### CODE REVIEW

**Comment by jovove** [comment]
Content changes:
- Navbar
- Height of the landing page
- Font size of the dropdown menus
Kapture 2019-11-26 at 8 46 40

**Comment by jovove** [comment]
Kapture 2019-11-26 at 8 49 07
Input fields changes, size of button changes, the navbar glitches out

**Comment by jovove** [comment]
image
This is different from the sketches, the headlines and texts align differently

**Comment by jovove** [comment]
image
Content differs from design in abstract

**Comment by jovove** [comment]
image
Designs have different boxes and text styling, which is the correct version?

**Comment by jovove** [comment]
image
The height in the content is different in your designs compared to your code?

File: aboveFold.css [hunk 1]
```diff
@@ -20,7 +32,6 @@
     z-index: 1;
     text-align: center;
     max-width: 880px;
-    max-height: 382px;
     padding-top: 40px;
```
Comment (jovove) [inline aboveFold.css]: You can combine all paddings in one line:
```suggestion
    padding: 40px 0 40px 52px;
```

File: aboveFold.css [hunk 2]
```diff
@@ -406,9 +424,8 @@
         padding-right: 40px;
         margin-left: auto;
```
Comment (jovove) [inline aboveFold.css:423]: All margins can be written as one line
```suggestion
        margin: 0 auto;
```

**Comment by jovove** [comment]
Is the `cursor: pointer;` for all the links?

**Comment by Hana-latif98** [comment]
> Is the `cursor: pointer;` for all the links?

I decided to add `cursor: pointer;` to all components that are clickable! Or should I remove it?

**Comment by Hana-latif98** [comment]
> Content changes:
> 
> * Navbar
> * Height of the landing page
> * Font size of the dropdown menus
>   Kapture 2019-11-26 at 8 46 40

I changed the height of the content when resized and I also changed the font size in the main function so that should be looking better now!

The navigation bar is coded as intended which is for the logo to be lifted above the text and the create account button moves above the login button. I can change this if it is not living up to a certain standard :)

**Comment by Hana-latif98** [comment]
> image
> This is different from the sketches, the headlines and texts align differently

This is also changed and I added a font-size of 16px to be more readable!

**Comment by Hana-latif98** [comment]
> image
> Content differs from design in abstract

I changed the code so that the corner of the images touch, however I didn't want the whole row to exceed over 1200px. This means the image is not as wide as the design, I think it looks better this way.

**Comment by jovove** [comment]
> > Content changes:
> > 
> > * Navbar
> > * Height of the landing page
> > * Font size of the dropdown menus
> >   Kapture 2019-11-26 at 8 46 40
> 
> I changed the height of the content when resized and I also changed the font size in the main function so that should be looking better now!
> 
> The navigation bar is coded as intended which is for the logo to be lifted above the text and the create account button moves above the login button. I can change this if it is not living up to a certain standard :)

- I disagree strongly with this layout. It is underwhelming and the amount of negative space makes the content to spread out and out of date.

- The navigation doesn't seem coordinated, you have left aligned, the logo being pushed by margin, a button with background color taking up space and the button with white background looks like regular text and does not create the pattern one might think.

- The advertising have jumped to the top now as well, then the whole concept of the building illustrations feel like they are not being used as their intended purpose and the advertising at the top takes away the focus from the main content.

- There are two different dividers with drop-shadow in place — (also there is no master version of the sketch file? There is only a branch - you need to have a master file for the design to make sure which version is the latest version to be approved).

- The mobile version of the site has a advertising that is odd, is that a approved google ads size? 
image

**Review by jovove — CHANGES_REQUESTED** [review]
Added some notes with some changes — this branch is becoming too large so probably need to fix those issues and open up a new branch to sort it out. The project is currently to large in the files to manage and needs a better structure.

**Comment by jovove** [comment]
image
Links are dissapearing

image
Different links when responsive? 

Kapture 2019-12-10 at 13 41 08
One advertising just disappears and what was previous 100% height decreases, could you explain your process? 

image
Images don't intersect precisely. Please take a look at that and document what might be the cause

image
Content does not align

Other:
- The rest of the ads have disappeared completely when going into smaller devices

**Comment by jovove** [comment]
image

What dimensions are the google ads? the 470x68 size is nowhere to be found on google's recommendations (reference: https://support.google.com/google-ads/answer/7031480?hl=en)

**Comment by Hana-latif98** [comment]
> image
> Links are dissapearing
These are meant to be removed and are moved down to the footer. I was struggling to make the space work so I moved it out completely as I have seen other websites do this too.
> 
> image
> Different links when responsive?
This has been changed so the header links are now in the slider!
> 
> Kapture 2019-12-10 at 13 41 08
> One advertising just disappears and what was previous 100% height decreases, could you explain your process?
Height has been changed now to keep the designs cohesive! When it comes to advertising I was struggling to get the adverts of that size into the small space so I decided to remove a few by process of elimination. This is only because I want the main function box to fit in the home screen, maybe I could move the adverts somewhere else, like between partitions (like blog posts do). 
> 
> image
> Images don't intersect precisely. Please take a look at that and document what might be the cause
This should be fixed now, didn't zoom in when checking it so missed the space between!
> 
> image
> Content does not align
I had to change the title to be able to make it align. Tried via sketch to make it work but struggling with this. If it's not preferred to shorten the title then I can take another look at it. Thought I'd give it a shot as it is now. 
> 
> Other:
> 
> * The rest of the ads have disappeared completely when going into smaller devices
This has to do with the screen size and me wanting to prioritise the main function box. Especially in the mobile version!

**Comment by Hana-latif98** [comment]
> image
> 
> What dimensions are the google ads? the 470x68 size is nowhere to be found on google's recommendations (reference: https://support.google.com/google-ads/answer/7031480?hl=en)

The ad itself is 468 x 60 (third from the bottom in the google size add link) but the div holding it is a little larger to create the illusion of a border :)

**Pull request merged**